.github/workflows/unit-tests-workflow-for-customImportMap.yml

```diff
@@ -0,0 +1,59 @@
+name: Unit tests workflow
+on:
+  push:
+    branches:
+      - "*"
```

Comment:
Nit: Do you want the workflow to run for every push to any branch in this repo? I'd restrict this to also be only on main.

Reply:
sure, updating it